pom.xml

```diff
@@ -9,7 +9,13 @@
     <name>mvnpm</name>
     <description>Maven on NPM</description>
     <url>https://mvnpm.org/</url>
-    
+    <repositories>
+        <repository>
+            <id>mvnpm.org</id>
+            <name>mvnpm</name>
+            <url>https://repo.mvnpm.org/maven2</url>
+        </repository>
+    </repositories>
```

Comment:
This should not be here (in the pom.xml) but rather in your settings.xml

Reply:
Yes I forgot to remove it :)